Examples for Choreo Connect relevant OAS definitions (#5138)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md

`x-wso2-basePath` is used to group all HTTP resources of the OpenAPI under a single context. Therefore, it will be exposed as a sub resource of a single API using Choreo Connect. This is how Choreo Connect logically separates one API from the other. The `x-wso2-production-endpoints` refers to the actual backend service implementation of the API. Therefore, the latter mentioned details are mandatory to expose an API using Choreo Connect.

> Also, you can define endpoint configurations globally using the `x-wso2-endpoints` extension as explained in [here]({{base_path}}/reference/vendor-extensions-catalog/#x-wso2-endpoints). Then, endpoints available under the `x-wso2-endpoints` can use with `x-wso2-production-endpoints`. For a detailed example, 

Reviewer: ```suggestion
Also, you can define endpoint configurations globally using the `x-wso2-endpoints` extension as explained [here]({{base_path}}/reference/vendor-extensions-catalog/#x-wso2-endpoints). Then, endpoints available under the `x-wso2-endpoints` can be used with `x-wso2-production-endpoints`. For a detailed example, 
```

        -   Application User
    -   However, if you want to expose API/ resource without security, you can also use the `x-wso2-disable-security` extension. Find more information about this extension from [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/disabling-security/#disabling-security).

>    You can find an example on how these OpenAPI extensions are used in [Open API definition](https://github.com/wso2/product-microgateway/blob/main/samples/openAPI-definitions/petstore_basic.yaml).

Reviewer: ```suggestion
   You can find an example on how these OpenAPI extensions are used in the [Open API definition](https://github.com/wso2/product-microgateway/blob/main/samples/openAPI-definitions/petstore_basic.yaml).
```
